workflow.txt

```diff
@@ -8,3 +8,12 @@ git merge my-solution
 fork on github
 git checkout master
 git push origin my-solution
+
+
+
+Functions for this Exercise
+fork on github
+git clone
+git add .
+git commit -m ""
+git push origin my-solution
```

Comment:
In order to be able to push the `my-solution` branch, you need to create it. That's what `git checkout -b my-solution` does. That should come right after `git clone`.